lib/modules/manager/terraform/index.ts

```diff
@@ -19,7 +19,7 @@ export const categories: Category[] = ['iac', 'terraform'];
 
 export const defaultConfig = {
   commitMessageTopic: 'Terraform {{depName}}',
-  managerFilePatterns: ['/\\.tf$/'],
+  managerFilePatterns: ['/\\.tf$/','/\\.tofu$/'],
```

Comment:
```suggestion
  managerFilePatterns: ['*.tf', '*.tofu'],
```
can you test this against a real repo? it'll be much simpler 


alternative 
```suggestion
  managerFilePatterns: ['**/*.tf','**/*.tofu'],
```